cluster/manifests/01-vertical-pod-autoscaler/recommender-deployment.yaml

```diff
@@ -16,13 +16,19 @@ spec:
     metadata:
       labels:
         application: vpa-recommender
-        version: patched-0.3.0-master-9
+        version: patched-0.3.0-master-12
         component: vpa
     spec:
       serviceAccountName: system
       containers:
       - name: recommender
-        image: registry.opensource.zalan.do/teapot/vpa-recommender:patched-0.3.0-master-9
+        image: registry.opensource.zalan.do/teapot/vpa-recommender:patched-0.3.0-master-12
+        args:
+        - --stderrthreshold=info
+        - --v=5
+        - --memory-saver
+        command:
+        - "./recommender"
```

Comment:
is the "." in front correct? 
It now depends on CWD and I would like to have it better absolute path instead.

Reply:
Made the change